Documentation~/workflow-create-polyshape.md

```diff
@@ -1,34 +1,29 @@
+![A Poly Shape is a 3D shape extruded from any 2D polygon](images/PolyShape_HeaderImage.png)
 
-![A Poly Shape is a 3-dimensional shape extruded from any 2-dimensional polygon](images/PolyShape_HeaderImage.png)
+To create a custom mesh:
 
-To define a custom Mesh:
+1. Use one of the following:
+    * In the **Scene** view > **Tools** overlay, select **Create PolyShape**.
+    * In the main menu, go to **Tools** > **ProBuilder** > **Editors** > **New Poly Shape**.
+1. To create the outer bounds of your mesh, click in the **Scene** view to create control points.
+    > **Note:** You can click directly on another surface to create an outgrowth, even when that surface is on the y-axis. 
+    ![Making a 2D shape on the wall](images/PolyShape_Draw1.png)
+1. To finish placing control points, press **Enter** or **Space**.
+    ![Extruding the 2D shape into a 3D Mesh](images/PolyShape_Draw2.png)
+1. To extrude a 3D shape, move your mouse up or down.
+1. To finish the shape, press **Enter** or **Space**. Your new mesh is now [in editing mode](polyshape.md) so you can continue to change it.
 
-1. Open the ProBuilder window (in Unity's top menu: **Tools** > **ProBuilder window**).
+![Finished Poly Shape](images/PolyShape_Draw3.png)
 
-  The [Edit Mode toolbar](edit-mode-toolbar.md) and the [ProBuilder toolbar](toolbar.md) appear.
+To edit a shape you created with the **Poly Shape** tool, in the **Scene** view:
 
-2. From the ProBuilder toolbar, click **New Poly Shape** (![Poly Shape Icon](images/icons/NewPolyShape.png)). 
+1. Select the shape.
+1. From the **Tools** overlay, select **Edit Poly Shape**.
 
-  > **Tip:** You can also access this tool from the ProBuilder menu (**Tools** > **ProBuilder** > **Editors** > **New Poly Shape**).
-
-3. In the Scene view, click to create control points. Control points form the outer bounds of your Mesh.
-
-	You can click directly on another surface to create an outgrowth, even when that surface is on the y-axis. 
-
-	![Making a 2D shape on the wall](images/PolyShape_Draw1.png)
-
-4. When you finish placing points, use either **Enter** (Windows), **Return** (macOS), or **Spacebar** to finalize the shape.
-
-	![Extruding the 2D shape into a 3D Mesh](images/PolyShape_Draw2.png)
-
-5. Move the mouse up and down to set the Mesh height. This extrudes a 3D Mesh from the 2D shape you defined with your control points.
-
-6. Click to exit create mode. Your new Mesh is now [in editing mode](polyshape.md) so you can continue to modify it.
-
-	![Finished Poly Shape](images/PolyShape_Draw3.png)
+> **Note**: When you modify control points or settings, you lose any changes you made in editing the mesh.
 
+[[Jon: I don't understand this note. What is the timeline for losing changes? As in, does it only happen if *while I'm editing* I also change control points or settings? How do I change control points or settings? Whose settings?]]
```

Comment:
Good question. I think it only happens when you're editing. @modrimkus is this true?

Reply:
@suearkinunity @JonMUnity The note is trying to say that regular mesh editing vs working procedurally with shapes don't really mix that well. You can't do:
1) procedural shape edits
2) direct vert/edge/face manipulation on top
3) procedural shape edits on top

You can do the first two steps and continue to say in low level manipulations but you can't return to the high/procedural level easily/without side-effects. 

When you try to do the 3rd step, prior to PB6, we'd remove the direct edits (2) and automatically put you back in base shape editing (1) and in PB6 there's now a warning shown and you have to "Reset shape" explicitly to go back to procedural editing.

https://github.com/user-attachments/assets/d7eadd8b-1fb6-457d-a3d9-216f5c7b53dc